templates/e2e-llm-workflows/README.ipynb

```diff
@@ -917,7 +941,8 @@
    ],
    "source": [
     "# Load test set for eval\n",
-    "ft_test_ds = ray.data.read_json(f\"{os.environ['ANYSCALE_ARTIFACT_STORAGE']}/viggo/test.jsonl\")\n",
+    "test_path = config_fields[\"test_path\"]\n",
+    "ft_test_ds = ray.data.read_json(test_path)\n",
```

Comment:
shouldn't this use datasets instead of the config fields?

Reply:
yeah, that's a good callout

i'll improve it to save the `anyscale.llm.dataset.Dataset` object from `anyscale.llm.dataset.upload()` to a var called `test_dataset`

```py
test_dataset = anyscale.llm.dataset.upload(...)
...
ft_test_ds = ray.data.read_json(test_dataset.storage_uri)
```